config/minion

```diff
@@ -0,0 +1,9 @@
+# we need to have different top file per formulas so call it with postfix formula
+# top-hana.sls , top-netweaver.sls etc etc
+
+state_top: top-hana.sls
```

Comment:
I think we don't need the `state_top` or `file_roots` entries here. They can be passed in the `salt-call` execution command

Reply:
state_top can't be passed

Reply:
resolved as comment